test-apps/crypto-trading/aggregates/src/main/java/org/elasticsoftware/cryptotrading/aggregates/orders/SellOrderProcess.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 - 2025 The Original Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package org.elasticsoftware.cryptotrading.aggregates.orders;
+
+import org.elasticsoftware.cryptotrading.aggregates.orders.commands.RejectOrderCommand;
+import org.elasticsoftware.cryptotrading.aggregates.orders.data.CryptoMarket;
+import org.elasticsoftware.cryptotrading.aggregates.orders.events.SellOrderRejectedEvent;
+import org.elasticsoftware.cryptotrading.aggregates.wallet.events.InsufficientFundsErrorEvent;
+import org.elasticsoftware.cryptotrading.aggregates.wallet.events.InvalidCryptoCurrencyErrorEvent;
+
+import java.math.BigDecimal;
+
+public record SellOrderProcess(
+        String orderId,
+        CryptoMarket market,
+        BigDecimal size,
```

Comment:
keep size and amount properties like in the BuyOrderProcess but set the amount to null in the constructor